projects/project1/Final Report/Students Infromation

```diff
@@ -0,0 +1,9 @@
+Group member: 1
```

Comment:
There is no need to share the student information on the repo. Please, consider removing the matriculation number at least!